sumatra/parameters.py

```diff
@@ -108,6 +108,9 @@ def parse_command_line_parameter(self, p):
     def diff(self, other):
         return _dict_diff(self, other)
 
+    def items(self):
```

Comment:
Why not as `@property`, so that one could access through `set.items` instead of `set.items()`.